<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainFilterDialog</class>
 <widget class="QDialog" name="mainFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>545</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Video Filter Manager</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QGridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="1" column="2" rowspan="4" colspan="2">
      <widget class="QListWidget" name="listWidgetAvailable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>340</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>12</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;big&gt;&lt;b&gt;Active Filters&lt;/b&gt;&lt;/big&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5" rowspan="4" colspan="11">
      <widget class="QListWidget" name="listWidgetActive">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>340</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="5" colspan="11">
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="2" colspan="2">
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;big&gt;&lt;b&gt;Available Filters&lt;/b&gt;&lt;/big&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="4" colspan="2">
      <widget class="QListWidget" name="listFilterCategory">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>340</height>
        </size>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>Transform</string>
        </property>
        <property name="icon">
         <iconset resource="filter.qrc">
          <normaloff>:/new/prefix1/common/ADM_icons/videoFilter/1.png</normaloff>:/new/prefix1/common/ADM_icons/videoFilter/1.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Interlacing</string>
        </property>
        <property name="icon">
         <iconset resource="filter.qrc">
          <normaloff>:/new/prefix1/common/ADM_icons/videoFilter/2.png</normaloff>:/new/prefix1/common/ADM_icons/videoFilter/2.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Colors</string>
        </property>
        <property name="icon">
         <iconset resource="filter.qrc">
          <normaloff>:/new/prefix1/common/ADM_icons/videoFilter/4.png</normaloff>:/new/prefix1/common/ADM_icons/videoFilter/4.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Noise</string>
        </property>
        <property name="icon">
         <iconset resource="filter.qrc">
          <normaloff>:/new/prefix1/common/ADM_icons/videoFilter/5.png</normaloff>:/new/prefix1/common/ADM_icons/videoFilter/5.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sharpness</string>
        </property>
        <property name="icon">
         <iconset resource="filter.qrc">
          <normaloff>:/new/prefix1/common/ADM_icons/videoFilter/3.png</normaloff>:/new/prefix1/common/ADM_icons/videoFilter/3.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Subtitles</string>
        </property>
        <property name="icon">
         <iconset resource="filter.qrc">
          <normaloff>:/new/prefix1/common/ADM_icons/videoFilter/7.png</normaloff>:/new/prefix1/common/ADM_icons/videoFilter/7.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OpenGl</string>
        </property>
        <property name="icon">
         <iconset resource="filter.qrc">
          <normaloff>:/new/prefix1/common/ADM_icons/videoFilter/gl.png</normaloff>:/new/prefix1/common/ADM_icons/videoFilter/gl.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Miscellaneous</string>
        </property>
        <property name="icon">
         <iconset resource="filter.qrc">
          <normaloff>:/new/prefix1/common/ADM_icons/videoFilter/6.png</normaloff>:/new/prefix1/common/ADM_icons/videoFilter/6.png</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>707</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPreview">
       <property name="text">
        <string>&amp;Preview</string>
       </property>
       <property name="icon">
        <iconset resource="filter.qrc">
         <normaloff>:/new/prefix1/common/ADM_icons/videoFilter/thumbnail.png</normaloff>:/new/prefix1/common/ADM_icons/videoFilter/thumbnail.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="icon">
        <iconset resource="filter.qrc">
         <normaloff>:/new/prefix1/common/ADM_icons/videoFilter/close.png</normaloff>:/new/prefix1/common/ADM_icons/videoFilter/close.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listFilterCategory</tabstop>
  <tabstop>listWidgetAvailable</tabstop>
  <tabstop>listWidgetActive</tabstop>
  <tabstop>pushButtonPreview</tabstop>
  <tabstop>buttonClose</tabstop>
 </tabstops>
 <resources>
  <include location="filter.qrc"/>
 </resources>
 <connections/>
</ui>
